### ISSUE TITLE
Missing attribute 'RAND_LIMIT' in modules Box2D

### ISSUE THREAD

**Issue by gonzaiva** [rlworkgroup/garage#250]
The following traceback is produced when running `examples/tf/trpo_cartpole.py`:
```
  File "examples/tf/trpo_cartpole.py", line 3, in <module>
    from garage.envs.box2d import CartpoleEnv
  File "/home/aigonzal/ivanWorkspace/garage/garage/envs/box2d/__init__.py", line 1, in <module>
    from garage.envs.box2d.box2d_env import Box2DEnv
  File "/home/aigonzal/ivanWorkspace/garage/garage/envs/box2d/box2d_env.py", line 9, in <module>
    from garage.envs.box2d.box2d_viewer import Box2DViewer
  File "/home/aigonzal/ivanWorkspace/garage/garage/envs/box2d/box2d_viewer.py", line 1, in <module>
    from Box2D import b2ContactListener
  File "/home/aigonzal/anaconda2/envs/garage/lib/python3.6/site-packages/Box2D/__init__.py", line 20, in <module>
    from .Box2D import *
  File "/home/aigonzal/anaconda2/envs/garage/lib/python3.6/site-packages/Box2D/Box2D.py", line 434, in <module>
    RAND_LIMIT = _Box2D.RAND_LIMIT
```
It seems the problem is solved by running:
`pip install box2d-py`

**Comment by ryanjulian** [comment]
This is documented in dockerfile/Dockerfile.ci. See the do

The solution is:
```sh
conda activate garage
pip uninstall -y Box2D Box2D-kengz box2d-py
pip install Box2D
```

See https://github.com/openai/gym/issues/100 and https://github.com/pybox2d/pybox2d/issues/82

**Comment by gonzaiva** [comment]
Yes, I just noticed with `grep -n -r "box2d-py"` in garage.

**Comment by ryanjulian** [comment]
We should probably keep this open until gym/box2d fix the upstream bug.

**Comment by gonzaiva** [comment]
Okay, in the meant time, the patch has been implemented in #258

**Comment by ryanjulian** [comment]
ref #319

**Comment by naeioi** [comment]
This issue is fixed in #335 . Closed.
